cmd/meroxa/root/apps/deploy.go

```diff
@@ -483,7 +483,7 @@ func (d *Deploy) validateLanguage() error {
 		d.lang = GoLang
 	case "js", JavaScript, NodeJs:
 		d.lang = JavaScript
-	case "py", Python:
+	case "py", Python3, Python:
```

Comment:
i think there's one more case in deploy....

Reply:
@janelletavares that should be normalized by http://github.com/meroxa/cli/blob/ba431e0c47de0dea6ea7d4b43b0258de8e224a67/cmd/meroxa/root/apps/deploy.go#L480-L492 previously!